ci/steps.lib.yml

```diff
@@ -104,24 +104,41 @@ config:
     LD: #@ compiler
     INCLUDE_SLOW_TESTS: #@ str(not quick).lower()
     RUN_NIGHTLY_TESTS: #@ str(is_nightly).lower()
+
+    #! Exercise 'make help' in quick tests mode, to ensure 'help' still works.
+    #@ if quick:
+    MAKE_HELP: help
+    #@ else:
+    MAKE_HELP: ""
+    #@ end
+
+    #@ if is_debug:
+    BUILD_MODE: "debug"
+    #! Add this, just to ensure that verbose build outputs feature doesn't regress
+    BUILD_VERBOSE: "1"
+    #@ end
+
     #@ if sanitize == "asan":
     DEFAULT_CFLAGS: "-fsanitize=address"
     DEFAULT_LDFLAGS: "-fsanitize=address"
+    BUILD_ASAN: "1"
     #! work around issue "LeakSanitizer has encountered a fatal error", may be kernel-dependent
     ASAN_OPTIONS: "detect_leaks=0"
     #@ elif sanitize == "msan":
     DEFAULT_CFLAGS: "-fsanitize=memory"
     DEFAULT_LDFLAGS: "-fsanitize=memory"
+    BUILD_MSAN: "1"
     #@ end
+
   run:
     path: sh
     dir: #@ input_name
     args:
     - "-c"
     #@ if is_debug:
-    - "make debug && ./test.sh"
+    - "make $MAKE_HELP debug all run-tests"
```

Comment:
This target will still need to change when @rtjohnso's PR lands, right?

Reply:
Well, @ajhconway -- Actually the hope was that for the initial merge of @rtjohnso 's `Makefile` rework, this target need not change.

Rob has added support for `make debug` as an acceptable target, so the thinking is that that PR will merge cleanly.

We will then have to come back and rip out these older targets, to smooth out the entire CI-usage. It's a multi-pass cleanup.